### ISSUE TITLE
Draggable views accessibility

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#9306]
# Problem
Form field missing a label for draggable views on https://nutrition.athletics.uiowa.edu/people. 

# Steps to reproduce
https://my2.siteimprove.com/Inspector/1189808/A11Y/Page?pageId=64171676474&impmd=85AD2477C69FE78057F1F127C9582636&issue=cantTell+failed&wcag=twopointtwo#/sia-r8/failed/Unp5NVA5YlI=;ZjEzNTA2MGI=

<img width="1166" height="722" alt="Image" src="https://github.com/user-attachments/assets/f30d068f-7017-44c5-a7f8-a3d502c08282" />

# Proposed solution

**Comment by cory-skeers** [comment]
This got raised up by CLAS as they're working through their sites: https://communicationstudies.uiowa.edu/people
